tiatoolbox/models/architecture/hovernet.py

```diff
@@ -368,10 +371,12 @@ def forward(self, imgs: torch.Tensor):
         This method defines how layers are used in forward operation.
 
         Args:
-            imgs (torch.Tensor): Input images, the tensor is in the shape of NCHW.
+            imgs (torch.Tensor):
```

Comment:
This may have conflicts as I have changed it in a different #299 

```suggestion
            input_tensor (torch.Tensor):
```